src/lib/list/selection-list.ts

```diff
@@ -335,6 +345,10 @@ export class MatSelectionList extends _MatSelectionListMixinBase implements Focu
     });
   }
 
+  ngOnChanges() {
+    this._stateChanges.next();
```

Comment:
Would it make sense to only do this for now in the `disabled` setter, or is there other state shared between the two components?

Reply:
Currently there is no other "state" that needs to be shared between child and parent.

Since we use the `mixinDisabled`, we can't customize the setter and notify the state change. As an alternative, we can just remove the mixin and implement the disabled property our self.

That way, we could even get rid of the `stateChanges` subject and just manually run something like `options.forEach(option => option._markForCheck()`. But maybe we will have things to share at some point. What do you prefer? (removing mixin seems to be the cleaner way for now)

Reply:
Yeah, I would go with removing the mixin for now.

Reply:
You can declare the `changes: SimpleChanges` parameter on the `ngOnChanges` method which will tell you whether `disabled` changed.